indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -640,11 +648,23 @@ class RHRegistrationsModifyStatus(RHRegistrationsActionBase):
     """Accept/Reject selected registrations."""
 
     def _process(self):
-        approve = request.form['flag'] == '1'
-        for registration in self.registrations:
-            _modify_registration_status(registration, approve)
-        flash(_("The status of the selected registrations was updated successfully."), 'success')
-        return jsonify_data(**self.list_generator.render_list())
+        approve = request.form.get('flag') == '1'
```

Comment:
I think this is probably a good time to split accept/reject into two separate RHs. Previously combining the two actions made sense because there was almost no difference in the code, but that's no longer the case now...